src/asm3/dbupdates/35012.py

```diff
@@ -0,0 +1,4 @@
+from asm3.dbupdate import execute
+
+execute(dbo, "INSERT INTO lksfieldlink (LinkType) VALUES (?)", [_("Person - Couple", dbo.locale),])
```

Comment:
How is this query working? It's not specifying an ID! It should be adding this linktype for ID 31. Also, you only need to use the dangling comma in tuples containing one item - it's not necessary with lists